web/src/main/java/org/hillview/InitialObjectTarget.java

```diff
@@ -30,18 +35,47 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 import java.util.logging.Level;
 import java.util.stream.Stream;
 
 public class InitialObjectTarget extends RpcTarget {
+    private static final int HILLVIEW_DEFAULT_PORT = 3569;
```

Comment:
Maybe these should be in a separate class HillviewService, part of the platform.